### ISSUE TITLE
Duplicate issues on `comment-no-empty` and `scss/comment-no-empty`

### ISSUE THREAD

**Issue by ybiquitous** [kristerkari/stylelint-scss#482]
Hi there,

When I read the `comment-no-empty` rule's document, I was a bit confusing.

## Reproduction

Run the following commands on your terminal:

```shell
# Install packages
echo {} > package.json
npm i stylelint stylelint-scss stylelint-config-recommended
#=> stylelint-scss@3.18.0
#=> stylelint-config-recommended@3.0.0
#=> stylelint@13.6.1

# Prepare config
cat <<EOF >.stylelintrc.yml
extends:
  - stylelint-config-recommended

plugins:
  - stylelint-scss

rules:
  scss/comment-no-empty: true
EOF

# Prepare test file
cat <<EOF >test.scss
a {
  /**/
  //
}
EOF

# Run
npx stylelint test.scss
#=>
#=> test.scss
#=>  2:3  ✖  Unexpected empty comment   scss/comment-no-empty
#=>  2:3  ✖  Unexpected empty comment   comment-no-empty
#=>  3:3  ✖  Unexpected empty comment   scss/comment-no-empty
```

## Suggestion

I was confused that **3** issues were reported because I expected **2** issues of `scss/comment-no-empty`.

I could fix it by turning off the [original `comment-no-empty` rule](https://github.com/stylelint/stylelint/blob/a8042e7192cad993081cf982901948dbd5004048/lib/rules/comment-no-empty/README.md) as follows:

```diff
extends:
  - stylelint-config-recommended

plugins:
  - stylelint-scss

rules:
+ comment-no-empty: null
  scss/comment-no-empty: true
```

I think useful if we would update the `comment-no-empty` document like [`at-rule-no-unknown`](https://github.com/kristerkari/stylelint-scss/blob/f54d9861e35891312bda98afe2404a993a4262e0/src/rules/at-rule-no-unknown/README.md):

```json
{
  "rules": {
    "comment-no-empty": null,
    "scss/comment-no-empty": true
  }
}
```

What do you think?

**Comment by kristerkari** [comment]
Makes sense!

**Comment by ybiquitous** [comment]
@kristerkari Thanks! 😄

**Comment by kristerkari** [comment]
Thank you @ybiquitous for the contribution!
